docs/en/functionalities.rst

```diff
@@ -0,0 +1,413 @@
+===========================================================
+
+With the aim of ensuring the principle of data portability, the User MUST have access to specific functionalities, particularly to: 
+
+- Request the backup and storage of Electronic Attestations of Attributes obtained through their Wallet Instance; 
+- Request the restore of their Electronic Attestations of Attributes on another Wallet Instance. 
+
+For further details and resources, please refer to www.wallet.gov.it. 
+
+Deactivation of the Wallet Instance 
+************************************
+
+The deactivation of the Wallet Instance is the functionality that makes the Wallet Instance inactive and therefore no longer operational. The deactivation process can be triggered by different actors depending on the circumstances, specifically: 
+
+- By the User, in cases such as: 
+
+	- The device has been lost or stolen; 
+	- The device has been compromised; 
+	- The device has been reset to factory settings. 
+
+- By an authorized third party, in cases such as: 
+
+	- The Wallet Solution no longer meets the minimum security requirements. 
+
+The User MUST have the ability to voluntarily deactivate their Wallet Instance through: 
+- The Wallet Instance itself; 
+- A Touchpoint (e.g., a website) provided by the Wallet Provider; 
+- The device’s app store, by uninstalling the Wallet Instance. 
+
+Below are the functional requirements to support the User Experience related to the deactivation of the Wallet Instance: 
+
+- The User MUST access their Wallet Instance using the previously configured unlock method or MUST Authenticate at the Touchpoint provided by the Wallet Provider; 
+- The User MUST select the Wallet Instance deactivation functionality; 
+- The User MUST be informed that deactivating the Wallet Instance will invalidate previously obtained Electronic Attestations; 
+- The User MUST confirm the action to proceed with deactivation, or MAY cancel the operation; 
+- The User MUST receive confirmation of successful deactivation; 
+- The User MUST be notified that the Wallet Instance is inactive when logging in again; 
+- The User MUST have the ability to reactivate the Wallet Instance by re-downloading the app from the app store (if uninstalled) and/or by following the activation process again. For further details, please refer to the `Activation of the Wallet Instance`_ section. 
+
+Once the Wallet Instance is reactivated, Electronic Attestations of Attributes can be re-obtained by starting the issuance or restore process again. For more details, please refer to sections `Issuance of Electronic Attestations of Attributes`_ and `Backup and Restore of Electronic Attestation of Attributes`_. 
+
+In case of errors, the User MUST receive clear and consistent messages that provide information and guidance for resolution. For more details, please refer to the `Error Management`_ section. 
+
+For further details and resources, please refer to www.wallet.gov.it. 
```

Comment:
I definitively can say that I don't like this sentece about `www.wallet.gov.it <https://www.wallet.gov.it>`_

I would go for providing details and resources within these technical specs aiming to reduce fragmentation
Furthermore, it doesn't make sense to generally point to a website without specifying the section and where the content is available.